### PR TITLE
Parse significant whitespace

### LINKED ISSUE #744: Whitespace only text nodes are ignored
#### Description
Whitespace only text nodes are ignored within a 'xml:space="preserve"' block, whereas Firefox, Chrome and Edge do render them.

#### Example data
Example SVG file:
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:lang="en" viewBox="0 0 100 100" preserveAspectRatio="xMinYMin">
	<rect x="0.0%" y="0.0%" width="99.5%" height="99.5%" fill="#FFFFFF" />
	<text xml:space="preserve">
		<tspan x="0em" dy="1em">A</tspan>
		<tspan x="0em" dy="1em"> </tspan>
		<tspan x="0em" dy="1em">B</tspan>
	</text>
</svg>
```

No whiteline is rendered for the second tspan-element:
actual

However, the above mentioned browsers render it similarly to this:
expected

#### Used Versions
Branch: master 27beb9
OS: Win 10
.NET Core: 3.1

### PR DESCRIPTION
**Pull request by Pim-Mostert**
## Reference Issue
Fixes #744 

#### What does this implement/fix? Explain your changes.
It configures the XmlTextReader to parse significant whitespace. For reference: [XmlTextReader.WhitespaceHandling Property](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmltextreader.whitespacehandling?view=netcore-3.1)
Note:
> Because the XmlTextReader does not have DTD information available to it, SignificantWhitepsace nodes are only returned within an xml:space='preserve' scope.

#### Any other comments?
I really don't know too much about either SVG or XML, so I hope this fix is appropriate.

### CODE REVIEW

File: Source/SvgDocument.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (H1Gdev) [inline Source/SvgDocument.cs:1]: It seems that the file encoding has changed.
Please apply setting of EditorConfig.

**Comment by H1Gdev** [comment]
Please add tests like [this PR](https://github.com/vvvv/SVG/pull/751/).

File: Source/SvgDocument.cs (hunk 1 above)
Comment (Pim-Mostert) [reply Source/SvgDocument.cs:1]: According to VS code, the file is encoded in UTF-8 with BOM and CRLF, as is specified in .editorconfig. Also, I compared the previous and new version of this file with a HEX editor and the first couple of rows are identical. To be honest, I have no idea why GitHub highlights the first line as a change.

**Comment by Pim-Mostert** [comment]
> Looks good! Can you please add a line in the release notes (doc/ReleaseNotes.md)?

Sure

**Comment by Pim-Mostert** [comment]
> Please add tests like [this PR](https://github.com/vvvv/SVG/pull/751/).

Sure, done!

File: doc/ReleaseNotes.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ The release versions are NuGet releases.
 * check if BaseUri is absolute (see [PR #738](https://github.com/vvvv/SVG/pull/738))
 
 ### Fixes
+* fixed the parsing of significant whitespace (see [#744](https://github.com/vvvv/SVG/issues/744) and [PR #744])(https://github.com/vvvv/SVG/pull/745))
```
Comment (H1Gdev) [inline doc/ReleaseNotes.md:11]: PR #745

File: Source/SvgDocument.cs (hunk 1 above)
Comment (H1Gdev) [reply Source/SvgDocument.cs:1]: BOM does not exist.

File: doc/ReleaseNotes.md (hunk 2 above)
Comment (H1Gdev) [reply doc/ReleaseNotes.md:11]: And remove close bracket.

`[PR #745](`

File: Source/SvgDocument.cs (hunk 1 above)
Comment (Pim-Mostert) [reply Source/SvgDocument.cs]: It should now be present

File: doc/ReleaseNotes.md (hunk 2 above)
Comment (Pim-Mostert) [reply doc/ReleaseNotes.md]: Fixed it - apologies for my sloppiness

File: doc/ReleaseNotes.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ The release versions are NuGet releases.
 * check if BaseUri is absolute (see [PR #738](https://github.com/vvvv/SVG/pull/738))
 
 ### Fixes
+* fixed the parsing of significant whitespace (see [#744](https://github.com/vvvv/SVG/issues/744) and [PR #745](https://github.com/vvvv/SVG/pull/745))
```
Comment (H1Gdev) [inline doc/ReleaseNotes.md:11]: By convention, the list seems to be in ascending order.
Please add it below the list.

**Pull request merged**